python/ray/data/_internal/logging_json.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
+disable_existing_loggers: False
+
+formatters:
+    ray:
+        class: ray._private.ray_logging.formatters.JSONFormatter
+
+filters:
+    console_filter:
+        (): ray.data._internal.logging.HiddenRecordFilter
+    core_context_filter:
+        class: ray._private.ray_logging.filters.CoreContextFilter
```

Comment:
should we also add this on the default logging? https://github.com/ray-project/ray/blob/e79bc615de5f74385769f98c8ef4ba419fe9e9a9/python/ray/data/_internal/logging.yaml#L10

Reply:
This should already be included in the default `logging.yaml` and the behavior should match the default for the new JSON one (we apply the hidden record filter to console logs, but not to file logs).